cmake/Modules/Packages/KOKKOS.cmake

```diff
@@ -10,6 +10,21 @@ endif()
 if(Kokkos_ENABLE_CUDA)
   message(STATUS "KOKKOS: Enabling CUDA LAMBDA function support")
   set(Kokkos_ENABLE_CUDA_LAMBDA ON CACHE BOOL "" FORCE)
```

Comment:
`Kokkos_ENABLE_CUDA_LAMBDA` is no longer an option since `v4.1.0`

Reply:
Removed